commit 16321d33d718fd8714155de34703a831ca9ef863 (refs/changes/85/385/1)
Author: Todd Whiteman <todd.whiteman@joyent.com>
Date:   2016-08-29T12:30:25-07:00 (3 years, 1 month ago)
    
    AGENT-1033: Remove /proc mounting from cn-agent chroot-gtar script

diff --git a/bin/chroot-gtar b/bin/chroot-gtar
index 4f6c525..2cd66f1 100755
--- a/bin/chroot-gtar
+++ b/bin/chroot-gtar
@@ -175,10 +175,8 @@ for lib in $libs; do
 done
 
 # Setup minimum fake filesystem.
-TO_UMOUNT="$TO_UMOUNT $CHROOTDIR/proc $CHROOTDIR/dev/fd"
-TO_REMOVE="$TO_REMOVE $CHROOTDIR/proc $CHROOTDIR/dev/fd"
-mkdir -p $CHROOTDIR/proc
-mount -F proc proc $CHROOTDIR/proc
+TO_UMOUNT="$TO_UMOUNT $CHROOTDIR/dev/fd"
+TO_REMOVE="$TO_REMOVE $CHROOTDIR/dev/fd"
 [[ ! -e $CHROOTDIR/dev ]] && TO_REMOVE="$TO_REMOVE $CHROOTDIR/dev"
 mkdir -p $CHROOTDIR/dev/fd
 mount -F fd fd $CHROOTDIR/dev/fd
